sql/table.sql

```diff
@@ -1192,6 +1192,8 @@ CREATE TABLE structural_variation_association (
 @column somatic                          Flags whether this structural variation is known to be somatic or not
 @column breakpoint_order                 Defines the order of the breakpoints when several events/mutation occurred for a structural variation (e.g. somatic mutations)
 @column length                           Length of the structural variant. Used for the variants with a class "insertion", when the size of the insertion is known.
+@column allele_freq                      The max allele frequency of the minor allele of this variant, as reported by gnomAD.
```

Comment:
The frequency is associated with this allele and this study - other studies apart from gnomAD may have data. A better comment would be something like 'The frequency reported for this allele in this study '

Reply:
Thanks. Changed description.